### PR TITLE
Fix: namespace problem in monitoring document

### PR DESCRIPTION
**Pull request by Gorgeous-Patrick**
I think that at this point, the document has completely given up teaching people to establish prometheus in a namespace. 
Fix:
1. Deleted '-n prometheus' param in the commands given completely.
2. \<xxx\> is considered a macro substitution in markdown and we did not define \<namespace\> macro. So it is ignored. I replaced it with \\\<namespace\\\> and added some comments about the usage.

### CODE REVIEW

File: support/monitoring/grafana/grafana.yaml [hunk 1]
```diff
@@ -5,5 +5,6 @@ datasources:
     - name: Prometheus
       type: prometheus
       url: http://prometheus-server.default.svc.cluster.local
+      # url: http://localhost:8080
```
Comment (ypkang) [inline support/monitoring/grafana/grafana.yaml]: Remove this commented line.

**Review by ypkang — CHANGES_REQUESTED** [review]
@Gorgeous-Patrick good changes! Just one tiny thing to fix. I left comment on the line.

File: support/monitoring/grafana/grafana.yaml (hunk 1 above)
Comment (Gorgeous-Patrick) [reply support/monitoring/grafana/grafana.yaml]: Resolved

**Review by ypkang — APPROVED** [review]
LGTM

**Pull request merged**